packages/devtools_app/lib/src/screens/vm_developer/vm_developer_common_widgets.dart

```diff
@@ -177,41 +177,50 @@ class RequestableSizeWidget extends StatelessWidget {
 /// Wrapper to get the name of an ObjRef depending on its type.
 String? _objectName(ObjRef? objectRef) {
   String? objectRefName;
-  if (objectRef == null) return null;
 
-  if (objectRef is ClassRef ||
-      objectRef is FuncRef ||
-      objectRef is FieldRef ||
-      objectRef is LibraryRef) {
+  if (objectRef is ClassRef || objectRef is FuncRef || objectRef is FieldRef) {
     objectRefName = (objectRef as dynamic).name;
+  } else if (objectRef is LibraryRef) {
+    objectRefName =
+        objectRef.name?.isEmpty ?? false ? objectRef.uri : objectRef.name;
```

Comment:
I'm not sure if we have a lint against this, but I generally wrap non-trivial conditions with parens to improve readability. Something like this:

```dart
objectRefName = (objectRef.name?.isEmpty ?? false) ? objectRef.uri : objectRef.name;
```

Reply:
It doesn't show in the dart analysis, but I'll add the parenthesis.